content/source/downloads.html.erb

```diff
@@ -74,4 +74,22 @@ show_notification: false
       </a>
     </div>
   </div>
+
+  <div class='cloud-learn-cta'>
+    <div class='top'>
+      <h3>Getting Started</h3>
+      <p>Terraform is distributed as a single binary. Install Terraform by unzipping it and moving it to a directory included in your system's <code>PATH</code>.</p>
+    </div>
+    <div class='contain'>
+      <div class='cloud'>
+        <img src='/assets/images/terraform-cloud-logo.svg' alt='HashiCorp Terraform Cloud Logo' />
+        <a href='https://app.terraform.io/signup/account?utm_source=terraform_io_download'>Get started with Terraform Cloud</a>
```

Comment:
```suggestion
        <a href='https://app.terraform.io/signup/account?utm_source=terraform_io_download'>Sign up for Terraform Cloud</a>
```

Instead of using the "Get Started" phrase for both application signup and tutorials, what about "Sign up" for the application and "Get Started" for the tutorial?